### DISCUSSION TITLE
How to use WEBSITE_REQUIRED parameter in docker?

### DISCUSSION THREAD

**Discussion by cankirca** [WeblateOrg/weblate#11830] [Q&A] [answered]
Hi,
 
 Ran Weblate with docker without issue, but can't disable WEBSITE_REQUIRED in project creation settings.
 1. I tried to add WEBSITE_REQUIRED = "false" to the settings-override.py, not worked.
 2. I've added WEBLATE_WEBSITE_REQUIRED: false to the docker-compose.override.yml file, not worked as well.
 
 Can you help me to solve that please?
 Thank you.

**Comment by nijel** [comment]
Did you restart the container after doing the changes?

**Comment by cankirca** [comment]
Hi,

Yes, I did docker compose down and docker compose up few times. Also,
contributer name is not working as well, I've edited from the user profile
for it.
Lastly, when I add values to docker-compose.override.yml by adding weblate_
at the top of each parameter, it didn't allow me to run docker, says not
supported.
 Example: weblate_website_title: xxx

Michal Čihař ***@***.***>, 12 Haz 2024 Çar, 09:14 tarihinde
şunu yazdı:

> Did you restart the container after doing the changes?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/WeblateOrg/weblate/discussions/11830#discussioncomment-9747334>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFE4NRWL2CS2ODDMHZ4UODZG7RNXAVCNFSM6AAAAABJFBHMJ6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TONBXGMZTI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Can K.

**Reply by nijel** [reply]
Maybe you are adding it in a wrong place of docker-compose.override.yml?

**Comment by cankirca** [comment]
Yes, I'm adding them to the docker-compose.override.yml file in /home/weblate-docker directory.
 I had to add site title setting to the settings-override.py file to change my site title. it works within settings-override.py file, but not working in docker-compose.override.yml file, not allow me to compile with docker compose up -d as well when the title parameter is found in docker-compose.override.yml.

**Reply by nijel** [reply]
How do you run docker compose? The `docker-compose.override.ym` is not loaded automatically, you need  to include `-f docker-compose.override.yml` on the command line.

**Comment by cankirca** [comment]
@nijel, I didn't include that file, because there is no info about this detail in https://docs.weblate.org/en/latest/admin/install/docker.html
 When I add site title to the docker-compose.override.yml file, it says not supported while running docker compose, after removing site title value, it is working, and I didn't use -f command at anytime :-)

**Reply by nijel** [reply, marked as answer]
Sorry, I mixed this up with a different question. The evnrionment variables have to be string, when you specify `false` it is parsed as boolean from YAML. Try following instead:

```yaml
WEBLATE_WEBSITE_REQUIRED: "false"
```

**Comment by cankirca** [comment]
Worked like a charm, thank you!
